azerothcore/azerothcore-wotlk
# fix(DB/creature) Slain Auchenai Warrior & Slain Sha'tar Vindicator

## Changes Proposed:
-  Pin the "1" bit on the flag of the "CALL_KILLEDMONSTER" action
-  Add more spawn of Slain Auchenai Warrior and Slain Sha'tar Vindicator (in total we have 15 Warrior and 13 Vindicator)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13670
- Closes https://github.com/chromiecraft/chromiecraft/issues/4385

## SOURCE:


## Tests Performed:

- 
- 


## How to Test the Changes:


1. .quest add 10913
2. .go cr 1977353
3. Check if the burned corpse gives you only one count for the quest
4. Look around and see in the corpse are spawned in addictional points

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
